src/pyb2b/xml/ForecastFlightPlanRequest.xml

```diff
@@ -0,0 +1,31 @@
+ <flig:FlightListByAerodromeRequest xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/"
+ xmlns:flig="eurocontrol/cfmu/b2b/FlightServices">
+    <!--Optional:-->
+    <endUserId>bcramet</endUserId>
+    <sendTime>{send_time:%Y-%m-%d %H:%M:%S}</sendTime>
+    <dataset>
+    <type>FORECAST</type>
+    </dataset>
+    <includeProposalFlights>true</includeProposalFlights>
+    <includeForecastFlights>true</includeForecastFlights>
+    <trafficType>DEMAND</trafficType>
+    <!--Optional:-->
+    <trafficWindow>
+    <wef>{day} 00:00</wef>
+    <unt>{day} 23:59</unt>
+    </trafficWindow>
+
+    <requestedFlightFields>icaoRoute</requestedFlightFields>
```

Comment:
Is this the whole list of fields or just the ones that are relevant for you?
If not, I would appreciate the other fields in comment for now :sweat:

Reply:
The fields that are relevant for us. I'm not sure where I can find the whole list but if I find it I'll add them commented.